### ISSUE TITLE
line numbers in code listings can't be disabled in 8.1.0

### ISSUE THREAD

**Issue by evgeni** [getnikola/nikola#3426]
### Environment

**Python Version:**
3.7.7

**Nikola Version:**
8.1.0

**Operating System:**
Fedora 32

### Description:
This started over at Tweitter (https://twitter.com/zhenech/status/1278613632725782528) and I was asked to report a bug here.

With the upgrade to 8.1, the code listings in my site (https://github.com/evgeni/die-welt.net) started to get line numbers and I can't figure out a way to revert back to the 8.0 behaviour. I wouldn't mind the numbers that much, but the way the tables get transferred to RSS clients make it really hard to read the code there.

To summarize the thread:
* using Markdown as input for my posts, usually with braced code blocks (but also tried normal indented ones as shown in the CodeHilite docs for "not numbered")
* setting `MARKDOWN_EXTENSION_CONFIGS = { 'codehilite': {'linenos': False, 'linenums': False} }` in `conf.py` didn't have an effect either
* 8.0.4 used to generate `<pre class="code literal-block"><span></span><code>…` whereas 8.1.0 started to create a `<table>…`

Thanks a ton for your support and Nikola!

**Comment by Kwpolska** [comment]
When upgrading to v8.1.1, if you were affected by this bug, run `nikola build -a` (or remove the `cache` folder) to update posts. (Running with `-a` is also a good idea after a release, even if no changes seem to apply.)

**Comment by evgeni** [comment]
Thanks @Kwpolska! 8.1.1 now again renders my page the way 8.0.4 did.
